### PR TITLE
Added banning and unbanning user

### PR DESCRIPTION
**Pull request by BobAesthetic**

### CODE REVIEW

File: backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/BanUser.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+using System;
+using LeanCode.CQRS;
+using LeanCode.CQRS.Security;
+
+namespace FurnitureShop.Core.Contracts.Web.Users
+{
+    [AuthorizeWhenHasAnyOf(Auth.Roles.User)]
```
Comment (aliaksei-liavonik) [inline backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/BanUser.cs:7]: User can ban user?

File: backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/UnbanUser.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+using System;
+using LeanCode.CQRS;
+using LeanCode.CQRS.Security;
+
+namespace FurnitureShop.Core.Contracts.Web.Users
+{
+    [AuthorizeWhenHasAnyOf(Auth.Roles.User)]
+    public class UnbanUser : IRemoteCommand
```
Comment (aliaksei-liavonik) [inline backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/UnbanUser.cs:8]: The same question

File: backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        {
+            this.userManager = userManager;
+            this.dbContext = dbContext;
+        }
+
+        public async Task ExecuteAsync(CoreContext context, BanUser command)
+        {
+            var user = await userManager.FindByIdAsync(command.UserId.ToString());
+            var claims = await userManager.GetClaimsAsync(user);
+            await userManager.RemoveClaimsAsync(user,claims);
```
Comment (aliaksei-liavonik) [inline backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs:33]: All claims is only this value? Is this safe to delete all claims?

File: backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/UnbanUserCH.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        {
+            this.userManager = userManager;
+            this.dbContext = dbContext;
+        }
+
+        public async Task ExecuteAsync(CoreContext context, UnbanUser command)
+        {
+            var user = await userManager.FindByIdAsync(command.UserId.ToString());
+            var claims = await userManager.GetClaimsAsync(user);
+            await userManager.RemoveClaimsAsync(user,claims);
```
Comment (aliaksei-liavonik) [inline backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/UnbanUserCH.cs:32]: The same question

**Review by aliaksei-liavonik — CHANGES_REQUESTED** [review]
Just not sure if it does not have some additional data inside claims && and roles should be changed to admin

File: backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs (hunk 3 above)
Comment (BobAesthetic) [reply backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs:33]: mamy tylko usera i admina nie ma żadnych innych

File: backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/BanUser.cs (hunk 1 above)
Comment (BobAesthetic) [reply backend/src/Core/FurnitureShop.Core.Contracts/Web/Users/BanUser.cs:7]: a rzeczywiscie nie zauważyłem

File: backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs (hunk 3 above)
Comment (BobAesthetic) [reply backend/src/Core/FurnitureShop.Core.Services/CQRS/Web/Users/BanUserCH.cs:33]: ale w sumie w razie czego poweidzmy że zrobie usuwanie tylko user claimu

**Review by aliaksei-liavonik — APPROVED** [review]

**Pull request merged**